src/components/Translation/index.js

```diff
@@ -40,15 +40,15 @@ class Translation extends Component {
     const { translation, index } = this.props;
     const lang = translation.languageName;
     const isArabic = lang === 'arabic';
-
+    const isRtl = (lang === 'arabic' || lang === 'persian' || lang === 'urdu' || lang === 'punjabi');
     return (
       <div
         id={`trans${index}`}
         className={`${styles.translation} ${isArabic && 'arabic'} translation`}
       >
         <h4 className="montserrat">{translation.resourceName}</h4>
         <h2
-          className={`${isArabic ? 'text-right' : 'text-left'} text-translation times-new`}
+          className={`${isRtl ? 'lang-dir-rtl' : 'text-left'} text-translation times-new`}
```

Comment:
I would say add language name as css class in parent div and control text alignment using css. 

```css
className={`${styles.translation} ${isArabic && 'arabic'} ${lang} translation`}
```
Using this approach, we can: 
- Configure language specific fonts ( like we've using nafees font for Urdu) 
- Control the alignment and maybe more in future  
- And this approach is DRY(we've tafisr and footnotes in multiple languages)  

We even don't need `text-left` or `text-right` classes here.

Reply:
Of course it's better